lib/elixir/pages/references/naming-conventions.md

```diff
@@ -124,3 +124,22 @@ When you see `length`, the operation runs in linear time ("O(n) time") because t
 Examples: `length/1`, `String.length/1`
 
 In other words, functions using the word "size" in its name will take the same amount of time whether the data structure is tiny or huge. Conversely, functions having "length" in its name will take more time as the data structure grows in size.
+
+### get, fetch, fetch!
+
+When you see the functions `get`, `fetch`, and `fetch!` for collections, you can expect the following behaviours:
+
+- `get` returns `nil` if the item is not present in the collection, or the requested item.
```

Comment:
Technically it returns a default, which by default is nil.

Reply:
Good point!